#pragma once

#include "proto/raft.pb.h"
#include "proto/block.pb.h"
#include "proto/tx_request.pb.h"
#include "proto/tx_operation.pb.h"
#include "proto/tx_request.pb.h"
#include "proto/log_entry.pb.h"
#include "proto/tx_request.pb.h"
#include "proto/command.pb.h"
#include "proto/replay.pb.h"
#include "proto/load.pb.h"
#include "proto/calvin.pb.h"
#include "proto/dlv.pb.h"
#include "proto/tx_op_type.h"
#include "proto/deadlock.pb.h"
#include "proto/tx_log.pb.h"
#include "proto/pb_json.h"
#include "proto/dependency_set.h"
#include "proto/panel.pb.h"
#include "proto/raft_test.pb.h"